docs/rpi_tcpip_link.md

```diff
@@ -0,0 +1,105 @@
+# longer range TCPIP via NRF24L01 for a pair of Raspberry PIs
+
+## Intro
+
+This tutorial is trying to use `NRF24L01` to create a TCP/IP link between two Raspberry Pi boards.
+Due to long range of NRFs, some of those have 1800 meter wireless range, it would be good to have a TCPIP link between two RPi via NRFs so it would be possible to have a TCPIP connection between two RPi in longer range. The onboard wifi of RPi cannot do long ranges like 50 meter even in clear sight. 
+
+![schematics](https://raw.githubusercontent.com/RF24/RF24Gateway/refs/heads/master/images/rpi_tcpip_link.png)
```

Comment:
This assumes the image already exists in upstream master branch.
```suggestion
![schematics](rpi_tcpip_link.png)
```
In this form, doxygen will pull the image named `rpi_tcpip_link.png` from the images folder automatically.